gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/testing/LocalPubsubHelper.java

```diff
@@ -35,6 +35,8 @@
  * A class that runs a Pubsub emulator instance for use in tests.
  */
 public class LocalPubsubHelper {
+
+  private final int port;
   private final LocalServiceHelper serviceHelper;
   private final List<String> gcloudCommand;
   private final URL emulatorUrl;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'emulatorUrl' could be replaced by a local variable.](https://www.codacy.com/app/mziccard/gcloud-java/file/2296608831/issues/source?bid=3123109&fileBranchId=3188953#l42)